components/soap/src/main/java/org/switchyard/component/soap/composer/SOAPContextMapper.java

```diff
@@ -90,11 +102,48 @@ public void mapFrom(SOAPBindingData source, Context context) throws Exception {
         if (source.getStatus() != null) {
             context.setProperty(HTTP_RESPONSE_STATUS, source.getStatus()).addLabels(SOAP_MIME_LABELS);
         }
+        
+        //Gets the propagation label from the service domain
+        String propagationLabel=null;
```

Comment:
See this and its subclasses to understand what is Label:
https://github.com/jboss-switchyard/switchyard/blob/master/core/api/src/main/java/org/switchyard/label/Label.java